app/controllers/assets_controller.rb

```diff
@@ -34,26 +34,27 @@ def latest
   def index
     @page_title = "All music by " + @user.name
 
-    if current_user_is_admin_or_owner?(@user)
-      @assets = @user.assets
-    else
-      @assets = @user.assets.published
-    end
-    @assets = @assets.recent.paginate(:per_page => 200, :page => params[:page])
+    @assets = if current_user_is_admin_or_owner?(@user)
+                @user.assets
+              else
+                @user.assets.published
+              end
+    @assets = @assets.recent.paginate(per_page: 200, page: params[:page])
```

Comment:
Same with this kind of indentation, kinda bums me out. The code here wasn't great, but I'd prefer it not to rewrite if/else blocks.